### DISCUSSION TITLE
Progress bar when uploading files

### DISCUSSION THREAD

**Discussion by Lachney** [pterodactyl/panel#2819] [Feature Requests]
In the beta version of Pterodactyl, I miss seeing the progress bar when uploading files.

In this new version, there's only a loading circle.

I have crap upload, so I would love to see the progress, lol.

**Comment by delije23** [comment]
Agreed 100%! I asked them few weeks ago,and so did some other people too.

**Comment by schrej** [comment]
There already is an issue for that: #2302
